packages/components/src/dropdown-menu/README.md

```diff
@@ -1,8 +1,60 @@
-# Dropdown Menu
+# DropdownMenu
 
-Dropdown Menu is a React component to render an expandable menu of buttons. It is similar in purpose to a `<select>` element, with the distinction that it does not maintain a value. Instead, each option behaves as an action button.
+The DropdownMenu displays a list of actions (each contained in a MenuItem, MenuItemsChoice, or MenuGroup) in a compact way. It appears in a Popover after the user has interacted with an element (a button or icon) or when they perform a specific action. 
```

Comment:
While it probably _should_ use `MenuItem`, `MenuItemsChoice`, and `MenuGroup` components, it doesn't currently, nor did at the time of this pull request. This confused me for a moment in my debugging of https://github.com/WordPress/gutenberg/issues/14754#issuecomment-480268587 .

Right now it's only rendering its own `IconButton` for each control.

https://github.com/WordPress/gutenberg/blob/45da3cf1948aa2723bfb649dae37287ca2495d64/packages/components/src/dropdown-menu/index.js#L74

I'll see if in the course of resolving #14754 that it could be updated to at least render `MenuItem`. I'm not really sure what combination of `controls` prop for this component would be expected to result in `MenuGroup` or `MenuItemsChoice` being rendered, however.

The way it's implemented currently, it does support grouping of controls, but notably lacks any way of expressing a label to use for a `MenuGroup`, so I don't know that it would be an appropriate use of that component.

```jsx
<DropdownMenu
	controls={ [
		[
			{
				title: 'Group 1, Control 1',
				icon: 'arrow-up-alt',
			},
			{
				title: 'Group 1, Control 2',
				icon: 'arrow-up-alt',
			},
		],
		[
			{
				title: 'Group 2, Control 1',
				icon: 'arrow-up-alt',
			},
			{
				title: 'Group 2, Control 2',
				icon: 'arrow-up-alt',
			},
		]
	] }
/>
```

From these other related component documentations, I also note the phrasing "are intended to be used in a `DropDownMenu`", which may be misleading in that they're used _by_ `DropdownMenu`, but the developer doesn't really have the choice of providing DropdownMenu with any of these components.